File: ast/statement/continue.go
package statement

import "github.com/panda-io/micro-panda/ast/ast"

type Continue struct {
	StatementBase
}

func (*Continue) Validate(c ast.Context) {
	/* if c.LoopBlock == nil {
		//TO-DO add check
		//c.Program.Error(con.Position, "invalid continue")
	}*/
}

File: MicroPanda/AST/Statement/Continue.cs
namespace MicroPanda.AST.Statement;

internal class Continue : Statement
{
    internal override void Validate(Context context)
    {
        // TO-DO add check
        // if (context.LeaveBlock == null)
        // {
        //     context.Program.Error(Position, "invalid continue");
        // }
    }
}